components/workouts/CreateWorkout.jsx

```diff
@@ -52,7 +52,7 @@ export default ({ navigation }) => {
       <AddFinishButton onPress={() => alert('Workout Created')} />
       {/* Finish Button will take u back to workouts and add workout to list */}
       {/* This is where a list of set workout details components will go */}
-      <AddCycleButton title="Exercise" size={18} onPress={() => alert('Add Exercies')} />
+      <AddCycleButton title="Exercise" size={18} onPress={() => navigation.navigate('Add Exercises')} />
```

Comment:
We want this to be a "popover" right? Which means we need to switch this to a full screen modal. I'm working on an example of this for the LogWorkouts component